python/GafferUSDTest/USDShaderTest.py

```diff
@@ -80,6 +82,7 @@ def __assertShaderLoads( self, name, expectedParameters, expectedOutputs ) :
 
 	def testLoadUsdPreviewSurface( self ) :
 
+		legacyUsdVersion = pxr.Usd.GetVersion()[1] <= 23
```

Comment:
Normally I'd say that you need to compare both major and minor versions to be safe here, to account for minor resetting when major bumps. But I have no idea what major even is in USD, since minor is the year.